pom.xml

```diff
@@ -21,7 +21,7 @@
     <java.version>1.7</java.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <oasp4j.version>2.2.0-SNAPSHOT</oasp4j.version>
+    <oasp4j.version>2.1.1</oasp4j.version>
```

Comment:
@anmaso , I have changed oasp4j.version to 2.3.0-SNAPSHOT and checked in this file.